src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,13 +8,23 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String EXAMPLE = "311, Clementi Ave 2, #02-25, 182938";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses should be in the format: BLOCK, STREET, UNIT NUMBER, POSTAL CODE";
+    public static final String ADDRESS_VALIDATION_REGEX = ".+,.+,.+,.+";
 
-    public final String value;
+    public String value;
```

Comment:
For better encapsulation you can use `private`, even better think whether it's possible to remove this variable or not.